### PR TITLE
PRIME 1319 Add the Remote User Site on the Enrollee's enrolment 

### PR DESCRIPTION
**Pull request by johnnyl6**

### CODE REVIEW

**Review by mtpultz — APPROVED** [review]
image

File: prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html [hunk 1]
```diff
@@ -394,9 +394,14 @@
...
 
       </ng-template>
-
+      <address>
+        {{ remoteAccessSite.site?.physicalAddress?.street | default }},
+        {{ remoteAccessSite.site?.physicalAddress?.city | default }}
+        {{ remoteAccessSite.site?.physicalAddress?.provinceCode | default }}
+        {{ remoteAccessSite.site?.physicalAddress?.postal | default }}
+      </address>
+      Vendor: {{ remoteAccessSite.site?.siteVendors[0]?.vendorCode | configCode: 'vendors' | default }}
```
Comment (hogmann) [inline prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html:403]: nvm i had an out of date branch, testing now whoops 👍

Comment (hogmann) [reply prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html:403]: cancel that still and error. You need an extra optional for that chaining to work.

```
remoteAccessSite.site?.siteVendors?.[0].vendorCode
```

**Review by hogmann — CHANGES_REQUESTED** [review]

File: prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html (hunk 1 above)
Comment (hogmann) [reply prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html:403]: How it is written you don't verify siteVendors is not undefined before you try and index into it. You need one more conditional to make sure it's not the case.

Comment (johnnyl6) [reply prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html:403]: > How it is written you don't verify siteVendors is not undefined before you try and index into it. You need one more conditional to make sure it's not the case.

array optional operator is not supported in template. Used alternative approach to check.

**Pull request merged**